### DISCUSSION TITLE
animate spin not works

### DISCUSSION THREAD

**Discussion by jamaluddinrumi** [tailwindlabs/tailwindcss#8483] [Help]
**What version of Tailwind CSS are you using?**

v3.0.24

**What build tool (or framework if it abstracts the build tool) are you using?**

------------------------------
- Operating System: `Darwin`
- Node Version:     `v16.15.0`
- Nuxt Version:     `2.16.0-27358576.777a4b7f`
- Package Manager:  `yarn@1.22.19`
- Builder:          `webpack`
- User Config:      `bridge`, `features`, `router`, `loadingIndicator`, `loading`, `ssr`, `target`, `publicRuntimeConfig`, `head`, `plugins`, `components`, `modern`, `render`, `buildModules`, `humansTxt`, `purgeCSS`, `googleAnalytics`, `pwa`, `robots`, `sitemap`, `fontawesome`, `modules`, `i18n`, `content`, `image`, `tailwindcss`, `generate`, `build`
- Runtime Modules:  `storyblok-nuxt@2.0.1`, `8n`, `@nuxt/content@1.15.1`, `@nuxt/image@0.6.2`, `@nuxtjs/sitemap@2.4.0`
- Build Modules:    `nuxt-webpack-optimisations@2.2.3`, `@nuxtjs/dotenv@1.4.1`, `@nuxtjs/svg@0.3.0`, `@nuxtjs/fontawesome@1.1.2`, `@nuxt/postcss8@1.1.3`, `@nuxtjs/tailwindcss@4.2.1`, `@nuxtjs/html-validator@0.7.0`, `@nuxtjs/robots@2.5.0`, `@nuxtjs/pwa@3.3.5`, `@nuxtjs/google-analytics@2.4.0`, `@luxdamore/nuxt-humans-txt@1.0.2`
------------------------------

**What version of Node.js are you using?**

v16.15.0

**What browser are you using?**

Opera, Chrome, Firefox, Safari

**What operating system are you using?**

macOS

**Reproduction URL**A

https://www.indomascot.com

**Describe your issue**

animate spin only works on Opera Desktop & Safari 

Works on Opera
![Screen Shot 2022-05-31 at 6 38 24 PM](https://user-images.githubusercontent.com/446780/171164915-2a3ed3ce-49fc-4e26-bc77-d32e75b42a8a.png)

Not works on Chrome, Firefox & Safari
![Screen Shot 2022-05-31 at 6 26 57 PM](https://user-images.githubusercontent.com/446780/171164003-cb76571b-c8d0-4004-9cdd-fe6d33133f53.png)

but, it all works on all mobile browsers

**Comment by thecrypticace** [comment]
Hey! So this isn't necessarily a problem with Tailwind itself but an incompatibility between FF and Chrome/Safari/Opera. The key issue here is that your config has `important: true` in it and FF treats important declarations as having precedence over animations while Chrome, Safari, Opera, Edge, etc… do not. Strangely enough I think FF is technically correct here but it's unexpected. You can't make keyframe properties important though, that doesn't change anything per the spec.

Unfortunately, to fix this in FF, the only approach is to not use `important: true`. You could still scope using `important: '#someid'`. I believe that would work fine.

**Reply by jamaluddinrumi** [reply]
hey, thank's for the reply

i'll try later

**Comment by jamaluddinrumi** [comment]
glad, accidentally, i fix this by reordering imported css files

_before_ 
```
<style lang="scss">
@import "~/assets/css/tailwind.css";
@import "~/assets/css/style.css";
@import "~/assets/css/featherlight.min.css";
@import "~/assets/css/wp-featherlight.css";
@import "~/assets/css/swiper.css";
@import "lite-youtube-embed/src/lite-yt-embed.css";
@import "~foundation-sites/scss/foundation";
```

_after_
```
<style lang="scss">
@import "foundation-sites/scss/foundation.scss";
@import "lite-youtube-embed/src/lite-yt-embed.css";
@import "~/assets/css/tailwind.css";
@import "~/assets/css/featherlight.min.css";
@import "~/assets/css/wp-featherlight.css";
@import "~/assets/css/swiper.css";
@import "~/assets/css/style.css";
```

**Comment by jamaluddinrumi** [comment]
after doing some upgrade, it's coming again

![Screen Shot 2022-09-18 at 2 09 56 PM](https://user-images.githubusercontent.com/446780/190890247-4cc58598-ac5a-4a10-a3bb-0ab5d6023220.png)

#### Environment
------------------------------
- Operating System: `Darwin`
- Node Version:     `v16.17.0`
- Nuxt Version:     `2.16.0-27720022.54e852f`
- Nitro Version:    `0.5.3`
- Package Manager:  `yarn@1.22.19`
- Builder:          `webpack`
- User Config:      `bridge`, `features`, `router`, `loadingIndicator`, `loading`, `ssr`, `target`, `publicRuntimeConfig`, `head`, `plugins`, `components`, `modern`, `render`, `buildModules`, `humansTxt`, `purgeCSS`, `googleAnalytics`, `pwa`, `robots`, `sitemap`, `fontawesome`, `modules`, `i18n`, `content`, `image`, `tailwindcss`, `generate`, `build`
- Runtime Modules:  `storyblok-nuxt@2.0.1`, `@nuxtjs/i18n@7.3.0`, `@nuxt/content@1.15.1`, `@nuxt/image@0.6.2`, `@nuxtjs/sitemap@2.4.0`
- Build Modules:    `nuxt-webpack-optimisations@2.2.5`, `@nuxtjs/dotenv@1.4.1`, `@nuxtjs/svg@0.3.0`, `@nuxtjs/fontawesome@1.1.2`, `@nuxt/postcss8@1.1.3`, `@nuxtjs/tailwindcss@4.2.1`, `@nuxtjs/html-validator@0.7.1`, `@nuxtjs/robots@2.5.0`, `@nuxtjs/pwa@3.3.5`, `@nuxtjs/device@2.1.0`, `nuxt-purgecss@1.0.0`, `@luxdamore/nuxt-humans-txt@1.0.2`
------------------------------

#### Environment
------------------------------
- tailwindcss@3.1.8
------------------------------

#### Browser
------------------------------
- FF 
- Chrome
------------------------------

#### Reproduction URL
------------------------------
- https://www.indomascot.com
------------------------------

**Reply by thecrypticace** [reply]
Yep, this Is because chrome finally updated to match spec behavior. Your only option here is to _not_ use `important: true` unfortunately.

**Reply by jamaluddinrumi** [reply]
thank's @thecrypticace
